Resources/Locale/en-US/administration/commands/change-cvar-command.ftl

```diff
@@ -0,0 +1,9 @@
+﻿cmd-changecvar-no-arguments = You must specify a cvar.
```

Comment:
Some minor spelling inconsistentcies, is all.

Reply:
I'll quickly fix them.